distributed/protocol/serialize.py

```diff
@@ -54,11 +54,15 @@ def dask_loads(header, frames):
     return loads(header, frames)
 
 
-def pickle_dumps(x):
+def pickle_dumps(x, context=None):
     header = {"serializer": "pickle"}
     frames = [None]
     buffer_callback = lambda f: frames.append(memoryview(f))
-    frames[0] = pickle.dumps(x, buffer_callback=buffer_callback)
+    frames[0] = pickle.dumps(
+        x,
+        buffer_callback=buffer_callback,
+        protocol=context["pickle-protocol"] if context else None,
```

Comment:
Should this be `_` or `-`. Just noting it is `_` above

Reply:
Thanks, done I think